clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/data/ServerDataHelper.java

```diff
@@ -62,7 +61,7 @@ public ServerDataHelper(String tenant) {
         ObjectMapper objectMapper = new ObjectMapper();
 
         HttpURLConnection urlConnection = (HttpURLConnection) new URL(findGaugeMetricsUrl).openConnection();
-        urlConnection.setRequestProperty(String.valueOf(TENANT_HEADER_NAME), tenant);
+        urlConnection.setRequestProperty("Hawkular-Tenant", tenant);
```

Comment:
While I do not expect the actual header name to change, why go this route here and in subsequent classes?